Hyperbolic_surface_triangulation_2/test/Hyperbolic_surface_triangulation_2/hs_test_isometry.cpp

```diff
@@ -0,0 +1,86 @@
+#include <CGAL/Hyperbolic_surface_traits_2.h>
+#include <CGAL/Complex_number.h>
```

Comment:
Only `#include` what is required, so that the testsuite can check that the required headers include everything they need.